content/sensu-go/5.21/reference/assets.md

```diff
@@ -91,16 +91,70 @@ For further examples of Sensu users who have added the ability to use a communit
 
 ### Asset path
 
-When you download and install an asset, the asset's local path on disk is exposed to asset consumers via the asset's environment variables.
+When you download and install an asset, the asset's local path on disk is exposed to asset consumers via environment variables.
 This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or using a custom function for token substitution, `assetPath`.
 
-For example, if you included a configuration file in the `include` directory of your asset `sensu-go-hello-world-0.0.2`, you can reference the asset from your handler in either of these ways:
+For example, if you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can reference the asset from your handler in either of these ways:
 
-- `$SENSU_GO_HELLO_WORLD_0_0_2/include/config.yaml`
-- `${{assetPath "sensu-go-hello-world-0.0.2"}}/include/config.yaml`
+- `$SENSU_PLUGINS_WINDOWS/include/config.yaml`
+- `${{assetPath "sensu-plugins-windows"}}/include/config.yaml`
```

Comment:
We should probably mention that the `assetPath` function will only be available where token substitution is available, which means handlers & mutators cannot use it at the moment.